### ISSUE TITLE
Header Row Z-index can Dominate Other Visual Elements like Toolbar on Scroll (solved by user)

### ISSUE THREAD

**Issue by L-K-Mist** [jspreadsheet/ce#608]
This one had me puzzled for a while and didn't see an existing issue that fitted my filter terms.

![z-index](https://user-images.githubusercontent.com/22072669/65581679-2e76ff00-df7c-11e9-8a75-8193d57e75fc.png)

Tried setting z-index in various places with no luck.

Then in jexcel/dist/jexcel.css I saw the issue:

```
.jexcel > thead > tr > td
{
    border-top:1px solid #ccc;
    border-left:1px solid #ccc;
    border-right:1px solid transparent;
    border-bottom:1px solid transparent;
    background-color:#f3f3f3;
    padding:2px;
    cursor:pointer;
    box-sizing: border-box;
    overflow: hidden;
    position: sticky;
    top: 0;
    ***z-index:2000***;
}
```
The trick therefore is to override this z-index in your app to something more sensible like:

```
.jexcel > thead > tr > td {
  z-index: 2;
}
```

Hope that helps someone with a similar issue.

**Comment by pphod** [comment]
Actually, this is not an error. There are several things you can do, one example is using tableOverflow to limit the spreadsheet size. Sticky headers are useful for long tables.

**Comment by L-K-Mist** [comment]
Absolutely. Had no doubt you had valid reasons.  Realise now; words like "to something more sensible" comes across as downright snooty.  Sorry for that.

More accurate to say "Override the z-index to be under what you want it under and over what you want it over."

May this be a reference for those who encounter the same thing.

Like in my case; the first thing I attempted was to increase the z-index of my toolbar; to something I considered high like 20.  When that didn't work, I started looking for solutions but didn't find mention even though surely someone else has had this happen.  

So at least now someone using a framework like Vuetify will find a quick solution if the header-row comes out in front of the app-bar on scroll.

Amazing the spreadsheet capabilities (and reactive data updates) you've pulled off with the standard table element.    

All the best,
Dylan
